ping.go

```diff
@@ -335,12 +335,19 @@ func (p *Pinger) run() {
 		}
 		if p.Count > 0 && p.PacketsRecv >= p.Count {
 			close(p.done)
-			wg.Wait()
+			safeWait(&wg, recv)
 			return
 		}
 	}
 }
 
+func safeWait(wg *sync.WaitGroup, ch chan *packet) {
+	if len(ch) > 0 {
+		<-ch
```

Comment:
why not process the packet instead of just dropping it? ie, instead of `<-ch` could we have this?:

```go
r := <-ch
p.processPacket(r)
```

Reply:
Yes, it's better to process it.